channels/yandex-alice/src/main/kotlin/com/justai/jaicf/channel/yandexalice/api/AliceBotRequest.kt

```diff
@@ -17,10 +18,12 @@ data class AliceBotRequest(
     val meta: Meta? = null,
     val session: Session,
     val request: Request? = null,
+    @SerialName("state")
+    val state: State? = null,
     @SerialName("account_linking_complete_event")
     val accountLinkingCompleteEvent: JsonObject? = null
 ): BotRequest {
-    override val clientId = session.userId
+    override val clientId = session.application.applicationId
```

Comment:
`clientId` must be unique for every user of the skill, not for a while application

Reply:
The new field contains the **same** value `user_id` contained. The new place of the value is more semantically correct representation of what the value actually means. `application_id` is changed when a user re-logins.